src/coreclr/vm/interpexec.cpp

```diff
@@ -1105,6 +1141,29 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                 memset(LOCAL_VAR(ip[1], void*), 0, ip[2]);
                 ip += 3;
                 break;
+            case INTOP_LOCALLOC:
+            {
+                size_t len = LOCAL_VAR(ip[2], size_t);
+                void* pMemory = NULL;
+
+                if (len > 0)
+                {
+                    pMemory = pThreadContext->frameDataAllocator.Alloc(pFrame, &len);
```

Comment:
```suggestion
                    pMemory = pThreadContext->frameDataAllocator.Alloc(pFrame, len);
```
It is not necessary to use the aligned length for memset. We just need to clear the memory that the user code asked for. We do not need to clear the extra padding.